bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/MagnitudeBand.scala

```diff
@@ -18,36 +22,40 @@ object MagnitudeBand {
-  case object _r extends MagnitudeBand("r", 630, "red", MagnitudeSystem.AB)
-  case object _i extends MagnitudeBand("i", 780, "far red", MagnitudeSystem.AB)
-  case object _z extends MagnitudeBand("z", 925, "near-infrared", MagnitudeSystem.AB)
-
-  case object U  extends MagnitudeBand("U", 365, "ultraviolet")
-  case object B  extends MagnitudeBand("B", 445, "blue")
-  case object G  extends MagnitudeBand("G", 477)
-  case object V  extends MagnitudeBand("V", 551, "visual")
-  case object UC extends MagnitudeBand("UC",610, "UCAC") // unknown FWHM
-  case object R  extends MagnitudeBand("R", 658, "red")
-  case object I  extends MagnitudeBand("I", 806, "infrared")
-  case object Z  extends MagnitudeBand("Z", 913)
-  case object Y  extends MagnitudeBand("Y", 1020)
-  case object J  extends MagnitudeBand("J", 1220)
-  case object H  extends MagnitudeBand("H", 1630)
-  case object K  extends MagnitudeBand("K", 2190)
-  case object L  extends MagnitudeBand("L", 3450)
-  case object M  extends MagnitudeBand("M", 4750)
-  case object N  extends MagnitudeBand("N", 10000)
-  case object Q  extends MagnitudeBand("Q", 16000)
-
-  case object AP extends MagnitudeBand("AP", None, Some("apparent"), MagnitudeSystem.VEGA)
-
-  val all: List[MagnitudeBand] =
-    List(_u, _g, _r, _i, _z, U, B, G, V, UC, R, I, Z, Y, J, H, K, L, M, N, Q, AP)
+  // Values for Sloan filters (u', g', r', i', z') taken from Fukugita et al. (1996)
+  case object _u extends MagnitudeBand("u", nm(  356), nm(  46), "UV",             MagnitudeSystem.AB)
+  case object _g extends MagnitudeBand("g", nm(  483), nm(  99), "green",          MagnitudeSystem.AB)
+  case object _r extends MagnitudeBand("r", nm(  626), nm(  96), "red",            MagnitudeSystem.AB)
+  case object _i extends MagnitudeBand("i", nm(  767), nm( 106), "far red",        MagnitudeSystem.AB)
+  case object _z extends MagnitudeBand("z", nm(  910), nm( 125), "near-infrared",  MagnitudeSystem.AB)
+
+  case object U  extends MagnitudeBand("U", nm(  360), nm(  75), "ultraviolet")
+  case object B  extends MagnitudeBand("B", nm(  440), nm(  90), "blue")
+  case object V  extends MagnitudeBand("V", nm(  550), nm(  85), "visual")
+  case object UC extends MagnitudeBand("UC",nm(  610), nm(  63), "UCAC")
+  case object R  extends MagnitudeBand("R", nm(  670), nm( 100), "red")
+  case object I  extends MagnitudeBand("I", nm(  870), nm( 100), "infrared")
+  case object Y  extends MagnitudeBand("Y", nm( 1020), nm( 120))
+  case object J  extends MagnitudeBand("J", nm( 1250), nm( 240))
+  case object H  extends MagnitudeBand("H", nm( 1650), nm( 300))
+  case object K  extends MagnitudeBand("K", nm( 2200), nm( 410))
+  case object L  extends MagnitudeBand("L", nm( 3760), nm( 700))
+  case object M  extends MagnitudeBand("M", nm( 4770), nm( 240))
+  case object N  extends MagnitudeBand("N", nm(10470), nm(5230))
+  case object Q  extends MagnitudeBand("Q", nm(20130), nm(1650))
```

Comment:
So we got rid of `G` and `Z`. Is it 100% confirmed that we don't use them? Are they on the old `Magnitude.Band`?